Adapter.C

```diff
@@ -66,6 +66,10 @@ bool preciceAdapter::Adapter::configFileRead()
             {
                 FFenabled_ = true;
             }
+            if (module == "Volume_Coupling")
+            {
+                Volume_Couplingenabled_  = true;
```

Comment:
Style-wise, the adapter uses nearly exclusively CamelCase, `VolumeCouplingEnabled_`. I guess it would be good to stick to it.